addons/popochiu/migration/helpers/popochiu_migration_helper.gd

```diff
@@ -0,0 +1,256 @@
+
+	if not dir.dir_exists(path):
+		return file_paths
+	
+	dir.list_dir_begin()
+	var element_name = dir.get_next()
+	while not element_name.is_empty():
+		var file_path := path.path_join(element_name)
+		if dir.current_is_dir():
+			if element_name in folders_to_ignore:
+				element_name = dir.get_next()
+				continue
+			
+			# Recurse into subdirectories
+			file_paths += get_absolute_file_paths_for_file_extensions(
+				file_path, file_extensions, folders_to_ignore
+			)
+		elif file_extensions.is_empty() or file_path.get_extension() in file_extensions:
+			# Add files with the specified extension to the [file_paths] array
+			file_paths.append(file_path)
+		
+		element_name = dir.get_next()
+	dir.list_dir_end()
+
+	return file_paths
+
+
+## Looks in the text of each file in [param file_paths] for coincidencies of [param from], and
+## replace them by [param to]. If any replacement was done, returns [code]true[/code].
+static func replace_text_in_files(from: String, to: String, file_paths: Array) -> bool:
+	return PopochiuUtils.any(
+		file_paths,
+		func (file_path: String) -> bool:
+			if not FileAccess.file_exists(file_path):
+				return true
+			
+			var file_read := FileAccess.open(file_path, FileAccess.READ)
+			var text := file_read.get_as_text()
+			file_read.close()
+			
+			if not from in text:
+				return false
+			
+			var file_write := FileAccess.open(file_path, FileAccess.WRITE)
+			text = text.replace(from, to)
+			file_write.store_string(text)
+			file_write.close()
+			return true
+	)
+
+
+## Returns [true] if the game is checked as pixel-art based on the value in
+## [code]popochiu/pixel/pixel_art_textures[/code] or in the old [code]popochiu_settings.tres[/code]
+## file in versions prior to [i]2.0.0-beta3[/i].
+static func is_pixel_art_game() -> bool:
+	var is_pixel_art := PopochiuConfig.is_pixel_art_textures()
+	
+	if FileAccess.file_exists(old_settings_file):
+		var old_settings := load(old_settings_file)
+		if old_settings.get("is_pixel_art_game") != null:
+			is_pixel_art = old_settings.is_pixel_art_game
+	
+	return is_pixel_art
+
+
+## Checks if [param text] exists in the text of the file at [param file_path].
+static func is_text_in_file(text: String, file_path: String) -> bool:
+	var file_read := FileAccess.open(file_path, FileAccess.READ)
+	var file_text := file_read.get_as_text()
+	file_read.close()
+	
+	return text in file_text
+
+
+## TODO: Document this function.
+static func replace_in_scripts(
+	replacements: Array[Dictionary], folders_to_ignore: Array[String] = []
+) -> bool:
+	var scripts_paths := get_absolute_file_paths_for_file_extensions(
+		PopochiuResources.GAME_PATH, ["gd"], folders_to_ignore
+	)
+	
+	var replaced_matches := 0
+	for dic: Dictionary in replacements:
+		replaced_matches += 1 if replace_text_in_files(dic.from, dic.to, scripts_paths) else 0
+	
+	return replaced_matches > 0
+
+
+## TODO: Document this function.
+static func get_rooms() -> Array[PopochiuRoom]:
```

Comment:
I also had my doubts when I left it here, but if you also think it could be useful in other scripts, I can move it to the `PopochiuEditorHelper` class.